src/player.js

```diff
@@ -707,6 +846,48 @@ export default class Player extends FakeEventTarget {
   }
 
   /**
+   * add off text track if there are actual text tracks associated with media
+   * setting this track is the same as calling Player's hideTextTrack
+   * @private
+   * @returns {void}
+   */
+  _addTextTrackOffOption(): void {
+    const textTracks = this.getTracks(TrackTypes.TEXT);
+    if (textTracks && textTracks.length) {
+      this._tracks.push(new TextTrack({
+        active: false,
+        index: textTracks.length,
+        kind: "subtitles",
+        label: "Off",
+        language: "off"
+      }));
+    }
+  }
+
+  _setDefaultTracks(){
+    const activeTracks = this.getActiveTracks();
+    const playbackConfig = this._config.playback;
+
+    if (playbackConfig.textLanguage == "auto"){
+      playbackConfig.textLanguage = Locale.language;
```

Comment:
Why do we change the user config? it will looks like a bug.
we should only do our inner logic according to the config which in this case is to send
`Locale.language` to `_setDefaultTrack ` .